Toggle comment: accept arg to bind shortcuts and customize comment behaviour (#3101)
BetterThanTomorrow/calva: CHANGELOG.md

## [Unreleased]

- Fix: [Toggling selection inside a bracketed form brakes structure](https://github.com/BetterThanTomorrow/calva/issues/3096)

> - Add: `calva.toggleLineComment` accepts a `behavior` argument, allowing keyboard shortcuts to be bound for any specific comment style independently of the setting (e.g. `"args": { "behavior": "commentCurrentLine" }`)

Reviewer: ```suggestion
- [Toggle comment: accept an argument to bind shortcuts for any of the supported behaviours](https://github.com/BetterThanTomorrow/calva/issues/3100)
```
